File: src/main/java/Objetos/Asociacion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Objetos;

/**
 *
 * @author james
 */
public class Asociacion {
    private String codigo;
    private String cliente;
    private String cuenta;
    private String clienteRe;

    public Asociacion(String codigo, String cliente, String cuenta) {
        this.codigo = codigo;
        this.cliente = cliente;
        this.cuenta = cuenta;
    }

    public Asociacion(String codigo, String cliente, String cuenta, String clienteRe) {
        this.codigo = codigo;
        this.cliente = cliente;
        this.cuenta = cuenta;
        this.clienteRe = clienteRe;
    }

    public Asociacion() {
    }

    
    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getCliente() {
        return cliente;
    }

    public void setCliente(String cliente) {
        this.cliente = cliente;
    }

    public String getCuenta() {
        return cuenta;
    }

    public void setCuenta(String cuenta) {
        this.cuenta = cuenta;
    }

    public String getClienteRe() {
        return clienteRe;
    }

    public void setClienteRe(String clienteRe) {
        this.clienteRe = clienteRe;
    }
    
    
    
    
}

File: src/main/webapp/Resourrcess/js/asociacion.js
/* 
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
window.onload = function () {
    $("#validarDatos").bind("submit", function () {
        var btnEnviar = $("#validar");
        $.ajax({
            type: $(this).attr("method"),
            url: $(this).attr("action"),
            data: $(this).serialize(),
            beforeSend: function () {
                btnEnviar.attr("disabled", "disabled");
            },
            complete: function (data) {
                btnEnviar.removeAttr("disabled");
            },
            success: function (data) {
                if (data === 'NOEXISTE') {
                    alert("No existe la cuenta que ingresaste");
                } else {
                    $("#busquedaCuenta").hide();
                    $("#datosBusqueda").fadeIn();
                }
            },
            error: function (data) {
                alert("Problemas al tratar de enviar el formulario");
            }
        });
        return false;
    });
};

function ocultarMostrar(ocultar, mostrar) {
    $("#validarDatos")[0].reset();
    bloquearBoton(document.getElementById("validar"));
    ocultar.hide();
    mostrar.fadeIn();
}

